[2024-12-03T14:02:33.249+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-03T14:02:33.321+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2024-12-03T14:02:33.354+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2024-12-03T14:02:33.355+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-03T14:02:33.759+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-02 00:00:00+00:00
[2024-12-03T14:02:33.909+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=217) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-03T14:02:33.921+0000] {standard_task_runner.py:72} INFO - Started process 226 to run task
[2024-12-03T14:02:33.927+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb', 'load_data', 'scheduled__2024-12-02T00:00:00+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb.py', '--cfg-path', '/tmp/tmp652ic_uo']
[2024-12-03T14:02:33.934+0000] {standard_task_runner.py:105} INFO - Job 178: Subtask load_data
[2024-12-03T14:02:40.299+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-02T00:00:00+00:00 [running]> on host ed9532b68cd8
[2024-12-03T14:02:40.750+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T00:00:00+00:00'
[2024-12-03T14:02:40.755+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-03T14:02:40.756+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-03T14:02:40.757+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-12-03 14:02:33.321944+00:00
[2024-12-03T14:02:40.758+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb and current dag run status:running
[2024-12-03T14:02:40.760+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-03T14:02:41.484+0000] {logging_mixin.py:190} INFO - Données chargées dans MongoDB, collection 'cleaned_data'
[2024-12-03T14:02:41.494+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-03T14:02:41.561+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-03T14:02:41.563+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb, task_id=load_data, run_id=scheduled__2024-12-02T00:00:00+00:00, execution_date=20241202T000000, start_date=20241203T140233, end_date=20241203T140241
[2024-12-03T14:02:41.655+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-03T14:02:41.656+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-03T14:02:41.657+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb queued_at:2024-12-03 13:56:36.463726+00:00
[2024-12-03T14:02:41.661+0000] {logging_mixin.py:190} INFO - Task hostname:ed9532b68cd8 operator:PythonOperator
[2024-12-03T14:02:41.874+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-03T14:02:41.903+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
